core/templates/self_list.h

```diff
@@ -77,7 +77,8 @@ class SelfList {
 		}
 
 		void remove(SelfList<T> *p_elem) {
-			ERR_FAIL_COND(p_elem->_root != this);
+			DEV_ASSERT(p_elem->_root == this);
```

Comment:
```suggestion
			ERR_FAIL_COND(p_elem->_root != this);
```
Restore this

Reply:
The suggestion after this removed it

Reply:
The suggestion after this did not change this, so you still need to fix this